pkgs/tools/video/xjadeo/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenv, autoreconfHook, fetchFromGitHub, ffmpeg, freetype, libGLU
+, libjack2, liblo, libX11, libXv, pkg-config, portmidi, xorg }:
+stdenv.mkDerivation rec {
```

Comment:
```suggestion
, libjack2, liblo, libX11, libXv, pkg-config, portmidi, xorg }:

stdenv.mkDerivation rec {
```

Reply:
It's a little unclear to me why there should be two newlines between a function's arguments and its body? Is this your opinion, or is this a formatting standard that's documented somewhere? `nixfmt` for example does not seem to mind about the lack of gap.

Reply:
If you take a look at other new packages you will see that this is standard formatting. Look at any good example at https://nixos.org/manual/nixpkgs/unstable/#how-to-develop and you will see that it is standard.

P.S.: It does not really matter what nixfmt says and if it minds the gap or not.